model/post.js

```diff
@@ -0,0 +1,25 @@
+const mongoose = require("mongoose");
+
+// schema for posting blog
+const postSchema = new mongoose.Schema({
+    title: {
+        type: String,
+        trim: true,
+        required: true
+    },
+    content: {
+        type: String,
+        trim: true,
+        required: true
+    },
+    comments: [{
+        type: mongoose.Schema.Types.ObjectId,
+        ref: 'Comment'
+    }],
```

Comment:
Remove this property.

Imagine if there are 1000 comments, this property would be holding 1000 Comment ids that would need to be populated, which is why you have `post: {ref: 'Post'}` on the `commentSchema`

Reply:
ok